src/components/layout/header/platform-switcher/index.tsx

```diff
@@ -1,13 +1,75 @@
+import React, { useState } from 'react';
```

Comment:
Don't need to import React

```suggestion
import { useState } from 'react';
```